packages/design-tokens/tokens/color/core.json

```diff
@@ -28,7 +28,7 @@
         "5": { "value": "#190f33" }
       },
       "green": {
-        "1": { "value": "#b4f1aa" },
+        "1": { "value": "#d1f2cb" },
```

Comment:
Out of curiosity, how did we land on the lighter shade?

Reply:
Personally I just adjusted the color locally until it felt right, then got @jeremycbloom's approval in the slack thread 🙂 I'll be opening a thread in #design-system to discuss the impact of the change in our Figma prototypes.